src/domain/core/src/entities/engine.rs

```diff
@@ -137,9 +137,13 @@ pub struct TransformResponseExt {
     pub new_offset_interval: Option<OffsetInterval>,
     /// Watermark advanced by the transaction, if any
     pub new_watermark: Option<DateTime<Utc>>,
+    /// Schema of the output
+    /// TODO: This field should be made required once all engines are updated
+    pub schema: Option<SchemaRef>,
```

Comment:
Let's be more explicit about the naming. The comment is saying "schema of the output", so maybe the field could be named `output_schema` to be expressive.

Reply:
done